### ISSUE TITLE
A11y: failed test on validation example

### ISSUE THREAD

**Issue by novikov82** [surveyjs/survey-library#10052]
https://surveyjstest.azurewebsites.net/form-library/examples/javascript-form-validation/reactjs
![Image](https://github.com/user-attachments/assets/c0c974ed-9005-40da-8ec3-241f1ee1ae0b)
